lib/internal/modules/cjs/loader.js

```diff
@@ -197,8 +197,7 @@ ObjectDefineProperty(Module, '_stat', {
 
 function updateChildren(parent, child, scan) {
   const children = parent?.children;
-  if (children && !(scan && ArrayPrototypeIncludes(children, child)))
-    ArrayPrototypePush(children, child);
+  if (children && !(scan && ArrayPrototypeIncludes(children, child))) { ArrayPrototypePush(children, child); }
```

Comment:
Can you keep line 201 as it is to limit the noise in git blame? Same for the other instances where there was two lines instead of one.

Reply:
This was all done by the lint fixer. I don’t want to spend the time manually going back and adjusting line breaks, but if you’d like to you’re welcome to push commits on this branch.

Reply:
```suggestion
  if (children && !(scan && ArrayPrototypeIncludes(children, child))) {
    ArrayPrototypePush(children, child);
  }
```